core/jvm/src/main/scala/scalaz/zio/App.scala

```diff
@@ -39,6 +39,8 @@ package scalaz.zio
  */
 trait App extends DefaultRuntime {
 
+  implicit val runtime: Runtime[Environment] = this
```

Comment:
I think maybe we should put this into the cats-interop package. That is, have an `App` that extends this app, inside cats-interop.

Then we could also provide other cats helpers there. What do you think?

Reply:
Yup, I think that'd be better.

Reply:
Any preferences on naming?

Reply:
`CatsApp`???

Reply:
I was expecting `CatzApp` :smile: but that's good, I just pushed :)